lib/plugins/aws/package/compile/functions/index.test.js

```diff
@@ -646,10 +646,7 @@ describe('AwsCompileFunctions', () => {
           Role: { 'Fn::GetAtt': ['IamRoleLambdaExecution', 'Arn'] },
           Runtime: 'nodejs12.x',
           Timeout: 6,
-          Tags: [
-            { Key: 'foo', Value: 'bar' },
-            { Key: 'baz', Value: 'qux' },
-          ],
+          Tags: [{ Key: 'foo', Value: 'bar' }, { Key: 'baz', Value: 'qux' }],
```

Comment:
I guess this changes are not intended (?)

Reply:
Yes, I left it in draft intentionally. This references a link on serverless/site which needs to be merged & deployed first. Once that goes live, I'll update this PR.